README.md

```diff
@@ -1,63 +1,64 @@
 
-For documentation and support, please visit the [JW Player Support Site](http://support.jwplayer.com/).
+## Example
 
-## Examples
-
-We have several articles and examples on the [JW Player Support Site](http://support.jwplayer.com/customer/portal/topics/564475-javascript-api/articles).
-
-The example below will find the element with an id of *myVideoId* and render a video player into it. We will then
-create an event handler to watch for when someone changes the volume.
+The example below will find the element with an id of *my_video* and render a video player into it. 
 
 ```js
-	jwplayer('myVideoId').setup({
+    // Create a jwplayer instance
+	jwplayer('my_video').setup({
 		file: '/uploads/example.mp4',
 	});
 
-	jwplayer('myVideoId').onVolume(function(event) {
-		console.log('The volume has changed', event);
+    // Add a custom callback for when user pauses playback
+	jwplayer('my_video').onPause(function(event) {
+        alert('Why did my user pause their video instead of watching it?');
 	});
 ```
 
-Note that we could also use functions **getVolume**, **setVolume** and others which can be found [here](http://support.jwplayer.com/customer/portal/topics/564475-javascript-api/articles).
-
-## Contributing
+Other callbacks that we provide include
+* **onComplete**
+* **onSeek**
+* **onVolume**
+* **[and more](http://support.jwplayer.com/customer/portal/topics/564475-javascript-api/articles)**
 
-### Style Guide
-For our javascript code, we follow the [Douglas Crockford Style guide](http://javascript.crockford.com/code.html)
+You also have the power to programatically set any configuration within the player. 
 
-Before submitting a change be sure to verify it follows our guide by using
-```sh
-grunt jshint
+```js
+    function bumpIt() {
+        player.setVolume( player.getVolume() + 10 );
+    }
 ```
 
-### Build Instructions
-
-To build the JW Player, you will need the following software:
-
- * Flex SDK 4.1: http://sourceforge.net/adobe/flexsdk/wiki/Downloads/
- * Ant 1.7.0: http://ant.apache.org/bindownload.cgi
-
-To compile with Flex and Ant, you'll first need to modify the `build.properties` file found in the `build` folder:
+## Contributing
 
-1. Set `flexsdk` to the install location of the Flex SDK (e.g. `/usr/local/bin/flex/`)
-1. Set `execextension` to `.exe` if you're using Windows; otherwise leave it blank.
+### Build Instructions
 
-You can now compile the player using Ant:
+ 1. Install [Node.js](https://nodejs.org/download)
+ 1. Install [Adobe AIR SDK](http://www.adobe.com/devnet/air/air-sdk-download.html)
+ 1. Download [player.swc 11.1](http://fpdownload.macromedia.com/get/flashplayer/installers/archive/playerglobal/playerglobal11_1.swc)
+ 1. Place the player.swc file into ```{AIRSDK_Compiler}/frameworks/libs/player/11.1/playerglobal.swc```
+ 1. Modify your flex-config.xml file to use target player 11.1

```

Comment:
You shouldn't need to modify flex-config.xml since the mxmlc argument -target-player=11.1 is used. @DonatoB was that an issue?